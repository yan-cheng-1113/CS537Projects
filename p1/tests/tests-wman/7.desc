No Man Page Entry - should exit returning 0 and print message
