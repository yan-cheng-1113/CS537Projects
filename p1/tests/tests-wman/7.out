No manual entry for pie
